### ISSUE TITLE
Screenreader issue for read-only text fields

### ISSUE THREAD

**Issue by sswweber** [vaadin/vaadin-text-field#493]
When using a screenreader for read-only text fields, the screenreader reads a bullet point to seperate label from textfield instead of the usual asterisk '*'. These bullet points are also read to users which have no meaning to them.
To mark the read-only attribute, vaadin uses an additional HTML5-Attribute '<aria-readonly>' which is never set to true. Therefore the attribute cannot be interpreted correctly by the screenreader.

**Comment by web-padawan** [comment]
> These bullet points are also read to users which have no meaning to them.

See also #221

**Comment by rolfsmeds** [comment]
Is this about _required_ fields (hence the bullet), or about _readonly_ fields as described?

**Comment by sswweber** [comment]
Yes, I meant the required fields for example in the Login Dialog. Sorry, if my initial description caused some confusion here.

**Comment by rolfsmeds** [comment]
Ok, so did I understand correctly that you're saying that the problem is twofold as follows?

1. the bullet is read as "bullet" by the screenreader (as described in #221) AND
2. the `required` attribute set on the `vaadin-text-field` is insufficient for the screenreader to interpret it as required (contrary to the comments in #221)?

**Comment by sswweber** [comment]
Yes, number 1. is true. 

For number 2. there is no problem as of now - after checking it again, I realised that the issue was on my end

**Comment by rolfsmeds** [comment]
Closing as duplicate of #221
